neo/Consensus/ConsensusService.cs

```diff
@@ -153,12 +157,23 @@ private void Log(string message, LogLevel level = LogLevel.Info)
 
         private void OnChangeViewReceived(ConsensusPayload payload, ChangeView message)
         {
-            if (context.State.HasFlag(ConsensusState.CommitSent))
+            if (message.NewViewNumber < context.ViewNumber)
+            {
+                // If we are at a higher view, we can send the regeneration msg.
+                Log($"send recovery from view: {message.ViewNumber} to view: {context.ViewNumber}");
+                localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeRecoveryMessage() });
```

Comment:
I think a good strategy here instead of having all consensus nodes respond, is to have the last 3 nodes that produced blocks respond. I can add a check here to see if this node is one of the last 3 validators.

@erikzhang What do you think of this strategy?